src/fiatconnect/ReviewScreen.tsx

```diff
@@ -0,0 +1,340 @@
+                currencyCode: cicoQuote.quote.fiatType,
+                exchangeRate: 1,
+              },
+            }}
+            testID="txDetails-fiat"
+          />
+        }
+      />
+      <LineItemRow
+        title={
+          <>
+            {`${tokenDisplay} @ `}
+            <CurrencyDisplay
+              amount={{ value: new BigNumber('1'), currencyCode: cicoQuote.quote.cryptoType }}
+              showLocalAmount={true}
+            />
+          </>
+        }
+        amount={
+          <TokenDisplay
+            amount={cicoQuote.quote.cryptoAmount}
+            tokenAddress={tokenAddress}
+            showLocalAmount={false}
+            testID="txDetails-crypto"
+          />
+        }
+        style={styles.sectionSubTextContainer}
+        textStyle={styles.sectionSubText}
+      />
+      {!!cicoQuote.fiatAccount.BankAccount?.fee && (
+        // TODO(any): consider using FeeDrawer if we want to show fee type / frequency
+        <LineItemRow
+          title={t('feeEstimate')}
+          amount={
+            <CurrencyDisplay
+              amount={{
+                value: 0,
+                currencyCode: cicoQuote.quote.fiatType,
+                localAmount: {
+                  value: cicoQuote.fiatAccount.BankAccount?.fee,
+                  currencyCode: cicoQuote.quote.fiatType,
+                  exchangeRate: 1,
+                },
+              }}
+              testID="txDetails-fee"
+            />
+          }
+          style={styles.sectionSubTextContainer}
+          textStyle={styles.sectionSubText}
+        />
+      )}
+    </View>
+  )
+}
+
+function PaymentMethod({
+  provider,
+  fiatAccount,
+  fiatAccountSchema,
+  fiatAccountLogo,
+}: {
+  provider: ProviderInfo
+  fiatAccount: any
+  fiatAccountSchema: FiatAccountSchema
+  fiatAccountLogo?: string
+}) {
+  const { t } = useTranslation()
+
+  // TODO(any): consider merging this with other schema specific stuff in a generic
+  // type and create via a factory
+  let displayText: string
+  switch (fiatAccountSchema) {
+    case FiatAccountSchema.AccountNumber:
+      const account: AccountNumber = fiatAccount
+      displayText = `${account.accountName} (...${account.accountNumber.slice(-4)})`
+      break
+  }
+
+  return (
+    <View style={styles.sectionContainer}>
+      <Text style={styles.sectionHeaderText}>{t('fiatConnectReviewScreen.paymentMethod')}</Text>
+      <View style={styles.paymentMethodContainer}>
+        <View style={styles.paymentMethodDetails}>
+          <View style={styles.sectionMainTextContainer}>
+            <Text style={styles.sectionMainText} testID="paymentMethod-text">
+              {displayText}
+            </Text>
+          </View>
+          <View style={styles.sectionSubTextContainer}>
+            <Text style={styles.sectionSubText} testID="paymentMethod-via">
+              {t('fiatConnectReviewScreen.paymentMethodVia', { providerName: provider.name })}
+            </Text>
```

Comment:
In the design, there's an estimated how long it takes for the fund to settle, is this a todo that we plan to add later? If so, can we add a todo here so we don't forget?

Reply:
Actually this component was based on the [Withdraw design](https://www.figma.com/file/bwIY7peeykI8K4wTrXs4iN/Fiat-Connect?node-id=1361%3A31757) which doesn't include the duration, the Add designs aren't final. I'll confirm with Nitya